website/docs/components/form/input-group/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,26 @@
+## Anatomy
+
+### Base component
+
+![Input Group Base Anatomy](/assets/components/form/input-group/input-group-base-anatomy.png =800x*)
+
+| Element | Usage |
+|---------|-------|
+| Input | Required; can be one of many types extended via the [Text Input](/components/form/text-input) component. |
+| Button | Required; can be either default or toggle which are extensions of the Helios [Button](/components/button) and [Toggle Button](/components/toggle-button) components. |
+| Container | Wraps the subcomponents in an auto layout container. |
+
+!!! Info
+
+The Input Group `Base` component is only available in Figma.
+!!!
+
+### Subcomponent anatomy
+
+The subcomponents contained within an Input Group are extensions of Helios form components and have the same anatomy as the master components they map to.
```

Comment:
We try to avoid using "master" within HashiCorp because of the negative connotation. Usually, I see "main" used instead.

```suggestion
The subcomponents contained within an Input Group are extensions of Helios form components and have the same anatomy as the main components they map to.
```

Reply:
Good catch, thanks for getting that